Makefile

```diff
@@ -88,8 +92,10 @@ CONCORD_BFT_CMAKE_FLAGS?= \
 			-DLEAKCHECK=${CONCORD_BFT_CMAKE_ASAN} \
 			-DTHREADCHECK=${CONCORD_BFT_CMAKE_TSAN} \
 			-DCODECOVERAGE=${CONCORD_BFT_CMAKE_CODECOVERAGE} \
-			-DTXN_SIGNING_ENABLED=${CONCORD_BFT_CMAKE_TRANSACTION_SIGNING_ENABLED}
-
+			-DTXN_SIGNING_ENABLED=${CONCORD_BFT_CMAKE_TRANSACTION_SIGNING_ENABLED} \
+			-DUSE_OPENSSL_SHA_256=${CONCORD_BFT_CMAKE_USE_OPENSSL_SHA_256} \
```

Comment:
as we see our #defines is:
#if defined USE_CRYPTOPP_HASH
using Digest = DigestHolder<CryptoppDigestCreator>;
#elif defined USE_OPENSSL_SHA_256
using Digest = DigestHolder<OpenSSLDigestCreator<SHA2_256> >;
#elif defined USE_OPENSSL_SHA3_256
using Digest = DigestHolder<OpenSSLDigestCreator<SHA3_256> >;
#endif

So if a macro is set to FALSE, is it defined or not?

Reply:
If all the three macros are set to false, then we get compilation stops during `make format` check.

Reply:
Addressed https://github.com/arc-vmware/concord-bft/pull/2/commits/be7b4d3f850359b235887ae82cbe8f4c29a080c4